CHANGELOG.md

```diff
@@ -1,3 +1,9 @@
+## [0.13.5] - 2020-09-18
+### Added
+- Added Support for 8BitDo NES & SNES Bluetooth wireless receivers
+	- Sony PS4 DS4 & Nintendo Switch Pro controllers tested and verified as working
+	- Note: 8BitDo receivers must be on Firmware v1.33 
+
```

Comment:
There have been no changes to this codebase (the compatibility is in the forked USB library), I'm not sure a change log entry is entirely appropriate.  How do others feel?

Reply:
True but the USB library is an inseparable part of this library, so changes in the former naturally affect the latter.

Otherwise, how do we handle changes in the USB library and notify people of them?